src/commands/archive-card.js

```diff
@@ -0,0 +1,178 @@
+"use strict";
+
+var List = require('term-list');
+  var prompt = require('prompt');
+
+var __ = function(program, output, logger, config, trello, translator) {
+
+    var trelloApiCommand = {};
+
+    trelloApiCommand.makeTrelloApiCall = function(options, onComplete) {
+        if (!options.match) {
+            options.match = 'exact';
+            logger.warning('--match parameter not specified, defaulting to exact title match');
+        }
+        logger.info("Adding card");
+
+        // Grab our boards etc
+        var boardId = translator.getBoardIdByName(options.board);
+        var listId = translator.getListIdByBoardNameAndListName(options.board, options.list);
+
+        // Get a list of cards on a board
+        trello.get("/1/boards/" + boardId + "/cards/visible", function(err, data) {
+
+            if (err) {
+                throw err;
+            }
+            if (data.indexOf("Trello is down") !== -1) {
+                logger.error("Trello is currently unavailable");
+                process.exit(1);
+            }
+
+            var foundCards = [];
+            for (var i in data) {
+                var card = data[i];
+
+                // We only want cards from the list we specified
+                if (card.idList != listId) {
+                    continue;
+                }
+
+                if (options.match == 'exact') {
+                    if (card.name == options.title || !options.title){
+                            foundCards.push(card);
+                    }
+                } else if (options.match == 'contains') {
+                    if (card.name.includes(options.title)) {
+                            foundCards.push(card);
+                    }
+                }
+            }
+
```

Comment:
I like your suggestion, however, I find it a bit disconcerting myself using 'contains' and not being able to confirm what will be actioned. eg, if I have a long list, it might accidentally match something I did not intend. Confirmation is then comforting :) Another option would be to add a optional flag, say '--confirm' , which would cause the interactive prompt, without it just go ahead.

 What do you think?

Just to note; delete-card also has the List that prompts the user when more than one match found, so breaking piping also.